src/main/java/gov/nih/nci/evs/api/configuration/FHIRConfig.java

```diff
@@ -34,4 +36,26 @@ public ServletRegistrationBean<HapiR4RestfulServlet> hapiR4() {
 
     return servletRegistrationBean;
   }
+
+  /**
+   * Hapi R5 servlet registration
+   *
+   * @return the servlet registration bean
+   */
+  @Bean
+  public ServletRegistrationBean<HapiR5RestfulServlet> hapiR5() {
```

Comment:
Any reason to create this in code instead of using dependency injection to create all objects?

Reply:
Not a clue honestly. It was modeled after what was done in the TermHub code for R4 and R5. @bcarlsenca Can you answer this quetion?